seqkit/cmd/translate.go

```diff
@@ -194,17 +194,13 @@ Translate Tables/Genetic Codes:
 
 				for _, frame = range frames {
 					_seq, err = record.Seq.Translate(translTable, frame, trim, clean, allowUnknownCodon, markInitCodonAsM)
-
-					if err != nil {
-						if skipTranslateErrors {
```

Comment:
@shenwei356 actually I just noticed you moved this back into the err if block, if that matches your style more feel free to just ignore and close this PR. I do think in this case the `checkError(err)` outside the if block is probably redundant then